doc/lib.rst

```diff
@@ -45,6 +50,8 @@ Automatic imports
   Nim message passing support for threads. **Note:** This is part of the
   system module. Do not import it explicitly. Enabled with `--threads:on`:option:.
 
+* `std/channels <channels.html>`_
+  Nim message passing support for threads when Arc/Orc memory management is used.  **Note:** This library is **not** automatically imported with the `--threads:on`:option:, and when it is used with that option the `Channels[T]` type must be aliased to refer to this library's implementation and not the the above library's automatically imported version.  The library API is similar but not identical to that of the above library in order to allow the new `Channel[T]` type to not require use as global variables but to allow easy passing of these channels between threads via atomic reference counting.
```

Comment:
```suggestion
* `std/channels <channels.html>`_
  Nim message passing support for threads with ARC/ORC memory management.
```
The rest of that explanation belongs to the `std/channels` documentation, if anything.

Also, this module isn't imported automatically, as you even note, so it makes no sense to put it in the "Automatic imports" category. "Core" or "Threading" would be more appropriate.

Reply:
Of course, you are absolutely correct that this should shortened and moved as you suggest; I got carried away in explaining the differences and use as compared to the built-in channels.

What category would you suggest, "Core" or "Threading"?

My feelings are mixed, as the module does not actually require `--threads:on` to work so it could be included in "Core" although it is then only of use as a kind of FIFO stack; it definitely will almost always be used with "Threading".

So this should be moved to "Threading"?